mutagen/aiff.py

```diff
@@ -39,8 +39,22 @@ class InvalidChunk(error):
 def is_valid_chunk_id(id):
     assert isinstance(id, text_type)
 
-    return ((len(id) <= 4) and (min(id) >= u' ') and
-            (max(id) <= u'~'))
+    if len(id) < 1 or len(id) > 4:
+        return False
+
+    for i in range(0, len(id)):
```

Comment:
This looks equivalent to the min/max code before. Why the rewrite?

Reply:
My assumption is that this code should check for a valid chunk ID (FOURCC).

Besides the length, the check was implemented if the character was higher then the tolerated character range  `min(id) >= u' '`. (Which is arguable as well, because the first character should not be a space because 1: padding of the ID is done at the end. 2: chunk-id is normalized to strip of spaces.)

Secondly it was checking on the last character for the upper limit `max(id) <= u'~'`. 

What about the characters in the middle?
What about the upper range of the first character, what about the lower range of the last character?

In the suggested change, it checks each character against the range.